Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
 
 ---
 
+### Date: 19 August 2019
+
+#### Who did you help this week?
+
+* Went over GitHub pull requests w YH
+* Ran PhD student office hours which I think are helpful - someone even came :laughing:
+
+#### Who helped you this week?
+
+* SDC made some awesome explaining GitHub slides from our conversation which are so freaking useful.
+  * http://doi.org/10.5281/zenodo.3369466
```

Comment:
I have read it.It is simple,clear and absolutely great!

Reply:
Nice work 👍